src/models/transfers.js

```diff
@@ -181,19 +181,23 @@ function validateIsAffectedAccount (account, transfer) {
   }
 }
 
-function validateAuthorizations (authorizedAccount, transfer, previousDebits) {
-  if (previousDebits && transfer.debits.length !== previousDebits.length) {
-    throw new UnprocessableEntityError('Invalid change in number of debits')
+/**
+ * @param {Account} authorizedAccount
+ * @param {Funds} funds
+ * @param {Funds|undefined} previousFunds
+ * @param {String} type "debit" or "credit"
+ */
+function validateAuthorizations (authorizedAccount, fundsList, previousFunds, fundsType) {
```

Comment:
I'm open to that, but its out of the scope of this PR -- the name 'funds' refers to the schema in five-bells-shared.